warehouse/logfield/logfield.go

```diff
@@ -21,4 +21,6 @@ const (
 	ErrorMapping          = "errorMapping"
 	DestinationCredsValid = "destinationCredsValid"
 	Provider              = "provider"
+	ManifestLocation      = "manifestLocation"
```

Comment:
If a field is rare, we expect it to be used only in one or two places we might not want to include it here to avoid pollution.